python/pyspark/cloudpickle.py

```diff
@@ -87,8 +87,8 @@
     PY2 = True
     PY2_WRAPPER_DESCRIPTOR_TYPE = type(object.__init__)
     PY2_METHOD_WRAPPER_TYPE = type(object.__eq__)
-    PY2_CLASS_DICT_BLACKLIST = (PY2_METHOD_WRAPPER_TYPE,
-                                PY2_WRAPPER_DESCRIPTOR_TYPE)
+    PY2_CLASS_DICT_SKIP_PICKLE_METHOD_TYPE = (PY2_METHOD_WRAPPER_TYPE,
+                                              PY2_WRAPPER_DESCRIPTOR_TYPE)
```

Comment:
Hey let's dont change this file but keep as is. It's the exact copy of cloudpickle release. We just port their fixes and dont have conflicts here for management purpose. Linters skip this file too and we had to put a lot of efforts to resolve the conflicts here before.

Reply:
I am going to upgrade this file at https://github.com/apache/spark/pull/29114. Seems that release version doesn't have this issue.

Reply:
ok so since you are upgrading, we are good to leave it as is, correct?  Your merge will fix it up.

Reply:
Yup, I think we're all good

Reply:
Thanks for addressing this @HyukjinKwon ! I avoided making changes in other files that I could tell were direct copies (e.g. stuff in `hive-thriftserver`) but did not realize this was one of those cases.